src/models/Company.js

```diff
@@ -12,11 +12,20 @@ const CompanySchema = new Schema({
     },
     contacts: {
         type: [String],
+        minlength: CompanyConstants.contacts.min_length,
+        maxlength: CompanyConstants.contacts.max_length,
     },
     bio: {
         type: String,
         maxlength: CompanyConstants.bio.max_length,
     },
+    hasFinishedRegistration: {
+        type: Boolean,
+        default: false
+    },
+    logo: {
+        type: String,
+    }
 });
 
 // Update offers from this company on name change
```

Comment:
Please make sure this hook is working. If not, either change the hook or use the findOneAndUpdate in the service `changeAttributes` function

Reply:
I can confirm that it does not trigger, so I changed the function used in `changeAttributes`